backend/src/main/java/de/bund/digitalservice/ris/norms/application/exception/AnnouncementNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package de.bund.digitalservice.ris.norms.application.exception;
+
+/** Indicates that the requested norm does not exist. */
```

Comment:
Typo: should say "announcement" instead of "norm"

Reply:
exactly. Good catch!

Reply:
exactly. Good catch!